components/core/src/os/process/mod.rs

```diff
@@ -22,19 +22,197 @@ mod windows;
 #[cfg(unix)]
 mod unix;
 
-#[cfg(windows)]
-pub use self::windows::{become_command,
-                        current_pid,
-                        handle_from_pid,
-                        is_alive,
-                        Pid};
-
 #[cfg(unix)]
 pub(crate) use self::unix::SignalCode;
 #[cfg(unix)]
 pub use self::unix::{become_command,
                      current_pid,
                      is_alive,
                      signal,
-                     Pid,
-                     Signal};
+                     Pid};
+#[cfg(windows)]
+pub use self::windows::{become_command,
+                        current_pid,
+                        handle_from_pid,
+                        is_alive,
+                        Pid};
+
```

Comment:
Delete this space?

Reply:
D'oh!

Reply:
Fixed: https://github.com/habitat-sh/habitat/pull/6450/commits/2d9a8a182abf655d03914dea1ffe9c7d97656584